core/ledger/kvledger/txmgmt/statedb/statecouchdb/statecouchdb.go

```diff
@@ -878,104 +926,119 @@ func applyAdditionalQueryOptions(queryString string, queryLimit int32, queryBook
 }
 
 type queryScanner struct {
-	namespace       string
-	db              *couchDatabase
-	queryDefinition *queryDefinition
-	paginationInfo  *paginationInfo
-	resultsInfo     *resultsInfo
-	exhausted       bool
+	namespace   string
+	db          *couchDatabase
+	queryDef    *queryDefinition
+	pageInfo    *paginationInfo
+	resultsInfo *resultsInfo
+	exhausted   bool
 }
 
 type queryDefinition struct {
-	startKey           string
-	endKey             string
-	query              string
-	internalQueryLimit int32
+	startKey string
+	endKey   string
+	query    string
+	limit    int32
 }
 
 type paginationInfo struct {
-	cursor         int32
-	requestedLimit int32
-	bookmark       string
+	cursor   int32
```

Comment:
Just like above, this is a misfit field in this struct. However, in this change, taking `paginationInfo` from outside while constructing a scanner gives a wrong impression that the caller is supposed to know what this field means.